### ISSUE TITLE
Running on several gpu 

### ISSUE THREAD

**Issue by SaraCasti** [daniilidis-group/neural_renderer#55]
Hello everyone!
I am working on a project which make use of your neural renderer implementation. We would like to run our code in parallel on several GPU. Running the code on gpu different from 0, we got an error, so we tried to pass device (properly modifing the source code of the renderer) forcing to work on gpu:1.  In this case, the renderer returns a tensor of 0. What could be the problem? Is mandatory to run the renderer on gpu:0 ?

Thanks so much in advance!

**Comment by zhipeng-fan** [comment]
Got the same problem here! Simply changing the device from cuda:0 to cuda:1 seems not going to work. Actually, if inspecting the the nvidia-smi very carefully, some of the operation were still carried out on GPU:0 even the device was changed to 1 somehow. Got all zero rendered image as well.

**Comment by SaraCasti** [comment]
> Got the same problem here! Simply changing the device from cuda:0 to cuda:1 seems not going to work. Actually, if inspecting the the nvidia-smi very carefully, some of the operation were still carried out on GPU:0 even the device was changed to 1 somehow. Got all zero rendered image as well.

Yes, exactly, we did the same, and we got the same result (all zero in the rendered image).. Do you find a way to fix it?
